markdown/chapters/meandeck/alternative-piles.md

```diff
@@ -117,26 +117,6 @@ As a result if they do cast an Endurance then you can pass and simply draw the
 second Oracle now and try to win with a Devotion of 4 with 3 cards remaining in
 your deck.
 
```

Comment:
We could leave this section in as a "play around Predict" example?

Reply:
Not really (or would require reworking the whole line) Ragavan can only disrupt at sorcery speed but Predict is harder to play around with the given pile 🤔